packages/web-app-search/src/index.ts

```diff
@@ -1,48 +1,49 @@
-import SearchBar from './portals/SearchBar.vue'
 import App from './App.vue'
 import List from './views/List.vue'
-import { Component } from 'vue'
 // eslint-disable-next-line @typescript-eslint/ban-ts-comment
 // @ts-ignore
 import translations from '../l10n/translations.json'
+import { ApplicationInformation, defineWebApplication } from '@ownclouders/web-pkg'
+import { extensions } from './extensions'
+import { extensionPoints } from './extensionPoints'
 
 // just a dummy function to trick gettext tools
 const $gettext = (msg: string) => {
   return msg
 }
 
-export default {
-  appInfo: {
-    name: $gettext('Search'),
-    id: 'search',
-    icon: 'folder'
-  },
-  routes: [
-    {
-      name: 'search',
-      path: '/',
-      component: App,
-      children: [
+const appInfo: ApplicationInformation = {
+  name: $gettext('Search'),
+  id: 'search',
+  icon: 'folder',
+  isFileEditor: false
+}
+
+export default defineWebApplication({
+  setup() {
+    return {
+      appInfo,
+      routes: [
         {
-          name: 'provider-list',
-          path: 'list/:page?',
-          component: List,
-          meta: {
-            authContext: 'user',
-            contextQueryItems: ['term', 'provider']
-          }
+          name: 'search',
+          path: '/',
+          component: App,
+          children: [
+            {
+              name: 'provider-list',
+              path: 'list/:page?',
+              component: List,
+              meta: {
+                authContext: 'user',
+                contextQueryItems: ['term', 'provider']
+              }
+            }
+          ]
         }
-      ]
+      ],
+      translations,
+      extensions: extensions(),
+      extensionPoints: extensionPoints()
     }
-  ],
-  translations,
-  mounted({
-    portal
-  }: {
-    portal: {
-      open: (toApp: string, toPortal: string, order: number, components: Component[]) => void
-    }
-  }): void {
-    portal.open('runtime', 'header', 1, [SearchBar])
   }
-}
+})
```

Comment:
Sweeeeet 😍 

Do we need to mention this in the changelog?

Reply:
Why in the changelog?

Reply:
The new search extension api